<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>curve_lines</class>
 <widget class="QWidget" name="curve_lines">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>813</width>
    <height>594</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="custom_graphics" name="curve_approx">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>391</width>
     <height>321</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>30</y>
     <width>160</width>
     <height>123</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="start_drag">
      <property name="text">
       <string>Drag</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="delete_progress">
      <property name="text">
       <string>Delete Line</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="confirm_button">
      <property name="text">
       <string>Confirm and Store</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="text_instructions">
      <property name="text">
       <string>Click to set Start Point</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>200</y>
     <width>237</width>
     <height>89</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="info_text">
      <property name="text">
       <string>Current Hold = 0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="line_info"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="next_line_button">
        <property name="text">
         <string>Next Line</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="storage_clear">
        <property name="text">
         <string>Delete Storage</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>custom_graphics</class>
   <extends>QGraphicsView</extends>
   <header>custom_graphics.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
